frontend/packages/dev-console/integration-tests/tests/serverless.scenario.ts

```diff
@@ -0,0 +1,134 @@
+import { execSync } from 'child_process';
+import { protractor, by, browser, ExpectedConditions as until } from 'protractor';
+import {
+  appHost,
+  checkLogs,
+  checkErrors,
+  testName,
+  retry,
+} from '../../../../integration-tests/protractor.conf';
+import {
+  installServerlessOperator,
+  firstKebabMenu,
+  machineSetsInput,
+  submitCount,
+  hardRefreshPage,
+} from '../views/serverless.view';
+import { switchPerspective, Perspective, sideHeader } from '../views/dev-perspective.view';
+import * as sidenavView from '../../../../integration-tests/views/sidenav.view';
+import * as crudView from '../../../../integration-tests/views/crud.view';
```

Comment:
nit: can we import only what's needed , any specific reason for importing everything

```
import { isLoaded, actionForLabel} from '../../../../integration-tests/views/crud.view'
```

it applies to all *.view imports across files

Reply:
We have to use this as the way I have used. You can check the other e2e tests of devconsole.

Reply:
am not saying that's not the way all am saying is it's nit , we can avoid importing everything when we need just one or two utils

Reply:
@gajanan194 why was this marked resolved?